### PR TITLE
Support disabling DEV_MODE on Docker

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Support disabling DEV_MODE on Docker
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | `docker-compose uo --build`
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27137)

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]
I approve but one day it could be cool to use `defines_custom.inc.php` which aim to do this instead of using sed

**Review by Progi1984 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @boubkerbribri

File: .docker/docker_run_git.sh [hunk 1]
```diff
@@ -30,6 +30,11 @@ fi
 # From now, stop at error
 set -e
 
+if [ $PS_DEV_MODE -ne 1 ]; then
+  echo "\n* Disabling DEV mode ...";
+  sed -ie "s/define('_PS_MODE_DEV_', true);/define('_PS_MODE_DEV_',\ false);/g" /var/www/html/config/defines.inc.php
```
Comment (SharakPL) [inline .docker/docker_run_git.sh:35]: @boubkerbribri just wondering, why did you use `-ie` flag which generates `defines.inc.phpe` file? Is this supposed to be a backup? What's the point of that?

Comment (boubkerbribri) [reply .docker/docker_run_git.sh:35]: I think it's an error that slipped away here

Comment (SharakPL) [reply .docker/docker_run_git.sh:35]: I'm making a PR for the config so I'll fix that too :)

**Pull request merged**